tools/generator/Tests/expected/InterfaceMethodsConflict/InterfaceMethodsConflict.xml

```diff
@@ -13,21 +13,28 @@
 		<method abstract="true" deprecated="not deprecated" final="false" name="close" native="false" return="void" static="false" synchronized="false" visibility="public">
 		</method>
 	  </interface>
-	  <class abstract="false" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object" 
+	  <class abstract="false" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object"
 	  		final="false" name="SomeObject" static="false" visibility="public">
 		<implements name="xamarin.test.I1" name-generic-aware="xamarin.test.I1" />
 		<implements name="xamarin.test.I2" name-generic-aware="xamarin.test.I2" />
 		<method abstract="false" deprecated="not deprecated" final="false" name="close" native="false" return="void" static="false" synchronized="false" visibility="public">
 		</method>
 	  </class>
-	  <class abstract="true" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object" 
+	  <class abstract="true" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object"
 	  		final="false" name="SomeObject2" static="false" visibility="public">
 		<implements name="xamarin.test.I1" name-generic-aware="xamarin.test.I1" />
 		<implements name="xamarin.test.I2" name-generic-aware="xamarin.test.I2" />
 		<!-- FIXME: this should not be required, but generator has an issue that when there is no member in the declaring class, "class_ref" is not generated and then the generated code does not compile because those invoker implementations depend on it. -->
 		<method abstract="false" deprecated="not deprecated" final="false" name="irrelevant" native="false" return="void" static="false" synchronized="false" visibility="public">
 		</method>
 	  </class>
+    <class abstract="false" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object"
```

Comment:
For good measure, add an `ITEM` constant field, just to see what happens!

Reply:
good call, added!

Reply:
done